modules/integration-tests/src/main/java/org/apache/fluo/integration/impl/FailureIT.java

```diff
@@ -132,8 +129,7 @@ private void testRollbackManyImpl(boolean killTransactor) throws Exception {
 
     if (killTransactor) {
       Stamp commitTs = env.getSharedResources().getOracleClient().getStamp();
-      exception.expect(FluoException.class);
-      tx2.commitPrimaryColumn(cd, commitTs);
+      Assert.assertThrows(FluoException.class, () -> tx2.commitPrimaryColumn(cd, commitTs));
```

Comment:
Used to this would throw an exception and nothing else in the method would run.  Now the code later in the method will run.  I am not sure if that matters, tomorrow I will look at the test in more depth to see what its doing.

Reply:
If it turns out that we do want to skip the rest, then it should be simple to do this to keep the same current behavior:

```suggestion
      Assert.assertThrows(FluoException.class, () -> tx2.commitPrimaryColumn(cd, commitTs));
      return;
```

Reply:
I looked at the test and its ok to run the code after the the assertThrows, I think its even an improvement.